app/Events/QueueMessageRequest.php

```diff
@@ -24,9 +25,10 @@ class QueueMessageRequest extends Event
      *
      * @return void
      */
-    public function __construct(Message $message)
+    public function __construct(Message $message, $sender)
```

Comment:
I was going to work on who to send after this so i can fix that up in a different pr